frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -285,12 +285,6 @@ class ProgramStructure::FixupExtern : public Modifier {
 };
 
 void ProgramStructure::createExterns() {
-    for (auto r : registers) {
```

Comment:
But also they do not seem to be converted to P4-16 data structures anywhere.
Where is that done?